packages/dataviews/src/components/dataviews-pagination/index.tsx

```diff
@@ -22,7 +22,9 @@ export function DataViewsPagination() {
 		paginationInfo: { totalItems = 0, totalPages },
 	} = useContext( DataViewsContext );
 
-	if ( ! totalItems || ! totalPages ) {
+	const isInfiniteScrollEnabled = view.infiniteScroll ?? false;
+
+	if ( ! totalItems || ! totalPages || isInfiniteScrollEnabled ) {
```

Comment:
Idea for a follow up: what do folks think about a go to top navigation? If you've got 100s of images, it might be helpful to go back to the top. I don't know, maybe it's just me 😄 

Another "me" thing is needing an idea of what "infinite" means, e.g., how many items are there.

Reply:
Yeah we could have a play with that! I'm not sure how it might work if the handler implementation doesn't just straight up load all items but instead loads next and unloads previous (which it might do for performance reasons, especially when catering to mobile devices). But def something to explore.